server/sonar-web/src/main/js/app/components/nav/component/ComponentNavBranch.tsx

```diff
@@ -130,32 +131,46 @@ export default class ComponentNavBranch extends React.PureComponent<Props, State
     const { configuration } = this.props.component;
     return this.state.dropdownOpen ? (
       <ComponentNavBranchesMenu
-        branches={this.props.branches}
+        branchLikes={this.props.branchLikes}
         canAdmin={configuration && configuration.showSettings}
         component={this.props.component}
-        currentBranch={this.props.currentBranch}
+        currentBranchLike={this.props.currentBranchLike}
         onClose={this.closeDropdown}
       />
     ) : null;
   };
 
   renderMergeBranch = () => {
-    const { currentBranch } = this.props;
-    if (!isShortLivingBranch(currentBranch)) {
+    const { currentBranchLike } = this.props;
+    if (isShortLivingBranch(currentBranchLike)) {
+      return currentBranchLike.isOrphan ? (
+        <span className="note big-spacer-left text-lowercase">
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([typescript:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3010&issues=AWIFEiOQJ6stENANvNAO&open=AWIFEiOQJ6stENANvNAO)

[ISSUE_KEY]: AWIFEiOQJ6stENANvNAO